docs/guide/install.rst

```diff
@@ -39,18 +39,14 @@ Homebrew installed, run the following:
 Windows 10
 ~~~~~~~~~~
 
-We recommend using `Anaconda <https://conda.io/docs/user-guide/install/windows.html>`_ for windows users.
+For a quick start you can move straight to installing Stable-Baselines in the next step (without MPI). This supports most but not all algorithms.
 
-0. Create a new environment in the Anaconda Navigator (at least python 3.5) and install ``zlib`` in this environment.
```

Comment:
This part is not needed anymore? I would still keep recommending anaconda anyway, no?

Reply:
`zlib` is part of Python by default, no? If this refers to some libraries, then those are not required. We have been able to do `pip install stable-baselines` on almost fresh Python installations on Windows.

Reply:
> is part of Python by default, no?

no idea... but good if it is not required

Reply:
I figure it referred to this package: https://docs.python.org/3/library/zlib.html